packages/block-editor/src/components/colors/utils.js

```diff
@@ -77,6 +77,9 @@ export function getColorClassName( colorContextName, colorSlug ) {
  */
 export function getMostReadableColor( colors, colorValue ) {
 	const colordColor = colord( colorValue );
-	return maxBy( colors, ( { color } ) => colordColor.contrast( color ) )
+	const getColorContrast = ( { color } ) => colordColor.contrast( color );
```

Comment:
Minor: In this case, we are computing the contrast and iterating on all the colors two times. I guess a simple reduce use previous value is { maxcontrast, color } would be more efficient and also simple.

Reply:
Thanks for the feedback 👍 

FWIW, I wanted to keep this simple, and I did previously try with a `reduce` approach first, but the code became quite longer, and I previously (on prior PRs) had received feedback that `reduce()` often makes code less readable. Then I saw the `colord` [benchmarks](https://github.com/omgovich/colord#benchmarks) and saw that it's extremely performant, so after confirming that the performance impact is negligible, I chose the shorter and more readable approach.